README.rst

```diff
@@ -88,6 +88,21 @@ You can also run your existing `unittest` tests with `marbles`
 
```

Comment:
Thank you for adding docs! Could we add this information to the [running tests](http://marbles.readthedocs.io/en/stable/quickstart.html#running-tests) section too?

Reply:
Yup! Thanks for point it out.